### DISCUSSION TITLE
I will not be around forever

### DISCUSSION THREAD

**Discussion by leppie** [leppie/IronScheme#153] [Announcements]
Yes, I will die at some point (not now, I am good).

As much as I love this project, I cant maintain it forever (but I'll try).

I would love someome to take it over.

**Comment by kant2002** [comment]
What's considerd maintainance in your opinion, and how do you preferred think project should move forward. I think that's would be implicitly asked in one way or another, and given you direct statement, it make take rounds on the internet. Would be great to have these things layed out upfront. Take care of youself.

**Comment by furesoft** [comment]
I would appreciate it to take it and continue developing and maintaining

**Comment by christoff-buerger** [comment]
@leppie : Your title! Really! I feelt sad a moment. Reading the rest, I recovered, but letting it settle, I am a bit crestfallen again.

**Reply by leppie** [reply]
Sorry. I have edited it a bit.

**Reply by christoff-buerger** [reply]
No, no, don't be sorry. I mean we don't know each other personally, and yet I felt sorry for you when I read the original title. You know, somehow it often feels that all the nice people that do soemthing good die off or resignate these days -- pushed out of motivation.

**Comment by GrumpyOldGandalf** [comment]
Hey @leppie - long time no chat.

I'd like to contribute and possibly consider assuming a maintainer role with IronScheme. How should I proceed?

Among other things, I note that Clojure CLR has implemented native compiling to disk on .NET Core 9.0+, and this is something that IronScheme wasn't even going to try to do until and unless MS added that functionality back. I'm not sure if you've considered revisiting that since.

**Reply by GrumpyOldGandalf** [reply]
BTW you may remember better as my other Apex-related Github account, bcalco.

**Reply by leppie** [reply]
I have implemented it for IronScheme too. Has been available since day 1 when .NET 9 got released. Just make sure you run on .NET 9. 

Also added native building (entire project) for Linux/Mac on .NET 9. See GH Actions for the workflow.
